django_cloud_deploy/cloudlib/service_account.py

```diff
@@ -120,10 +120,12 @@ def create_service_account(self, project_id: str, service_account_id: str,
                     format(service_account_id, response))
         except errors.HttpError as e:
             if e.resp.status == 409:
-                raise ServiceAccountCreationError(
-                    'Service account {} already exists within project {}'.
-                    format(service_account_id, project_id))
-            elif e.resp.status == 400:
+                # A service account with the same name already exist. This is
```

Comment:
I think its worth telling them that this is most likely because the tool was already run and has created these service accounts. Eg: 

Most likely our tool has created the service account already and is being re-run. This error code is fine as we just need the service accounts to exist. Later on we assign the appropriate roles for each one.

Reply:
Done. Thanks very much for rewording my comment!